lib/src/base-styles/_nx-divider.scss

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+.nx-divider {
+  background-color: var(--nx-swatch-indigo-90);
+  border-style: none;
+  margin: 24px 0;
```

Comment:
Just wondering if we should be using space variable like `var(--nx-spacing-6x)` here?

Reply:
> var(--nx-spacing-6x)

Added here: 00ac26997d924478e12602e8b0eaa42dca71be95